app/Traits/HasApproval.php

```
@@ -252,4 +252,20 @@
         $this->request_status = RequestStatuses::VOIDED;
         $this->save();
     }
+
+    public function notifyNextApprover($notificationModel)
+    {
+        $nextApproval = $this->getNextPendingApproval();
+        if ($nextApproval) {
+            $user = User::find($nextApproval['user_id']);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\User' in method 'notifyNextApprover'. (undefined)

(StaticAccess)